src/Appwrite/Platform/Tasks/Migrate.php

```diff
@@ -29,7 +30,8 @@ public function __construct()
             ->inject('cache')
             ->inject('dbForConsole')
             ->inject('getProjectDB')
-            ->callback(fn ($version, $cache, $dbForConsole, $getProjectDB) => $this->action($version, $cache, $dbForConsole, $getProjectDB));
+            ->inject('register')
```

Comment:
We can revert the changes here

Reply:
Keeping it as it is required for setting PDO